### PR TITLE
Add warehouse to Stock, apply changes, fix tests

### PR DESCRIPTION
**Pull request by badekk**
## Add warehouse to Stock, apply changes, fix tests
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

- [ ] Unit test(s)
- [x] Integration test(s)
- [x] Tested manually

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings
- [ ] I have added frontend code and checked it for mobile & desktop

### CODE REVIEW

**Comment by mar-pan** [comment]
Task linked: [CU-862jeuwcf Return stocks for user warehouse only](https://app.clickup.com/t/862jeuwcf)

File: src/backend/Order/Actions/StocksActions/StocksGetList/StocksGetListItemResponse.cs [hunk 1]
```diff
@@ -6,5 +6,6 @@ public class StocksGetListItemResponse
         public string ProductName { get; set; } = default!;
         public int Quantity { get; set; } = default!;
         public string Unit { get; set; } = default!;
+        public string WareHouseId { get; set; } = default!;
```
Comment (h4570) [inline src/backend/Order/Actions/StocksActions/StocksGetList/StocksGetListItemResponse.cs:9]: `WarehouseId`, to co napisałeś zostałoby prztłumaczone na json `wareHouseId`

**Review by h4570 — APPROVED** [review]
popraw i merguj

File: src/backend/Order/Actions/StocksActions/StocksGetList/StocksGetListItemResponse.cs (hunk 1 above)
Comment (badekk) [reply src/backend/Order/Actions/StocksActions/StocksGetList/StocksGetListItemResponse.cs]: fixed

**Pull request merged**